### ISSUE TITLE
[MacOS Bug] When you close the window of Upscayl you can't open it

### ISSUE THREAD

**Issue by Mikasa-san** [upscayl/upscayl#471]
### Checklist

- [X] I have checked that this issue isn't addressed in [the FAQ](https://github.com/upscayl/upscayl#-faq).
- [X] I have checked that this issue isn't addressed in any other issue or pull request.
- [ ] I have tried [setting my GPU ID](https://github.com/upscayl/upscayl/wiki/Guide#gpu-id).

### Paste the logs

```sh
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"General Photo (Ultrasharp)","value":"ultrasharp"}
⚙️ Setting gpuId to empty string
🔀 Setting model to 
🔄 Resetting image paths
🔀 Setting model to ultrasharp
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"General Photo (Ultrasharp)","value":"ultrasharp"}
⚙️ Setting gpuId to empty string
⚙️ Setting rememberOutputFolder to false
⚙️ Setting quality to 100%
```

### Describe the Bug

When you close the window of Upscayl without quit the app you can't create new window to use it

### To Reproduce

Open Upscayl then just close the window without quit Upscayl

### Version/Commit Hash

2.8.1

### OS Specifics

Mac Mini M1 - macOS Sonoma 14.0

### Expected Behavior

Upscayl should quit the app when we close the window or when click on it again it should create new window

### Screenshots

https://streamable.com/c2y95m

**Comment by NayamAmarshe** [comment]
Aha, thanks for reporting. I know exactly how to fix this one, I did include it in v2.8 but seems like I removed it later before release 😅

**Comment by NayamAmarshe** [comment]
Fixed in v2.8.5
